proto/api.proto

```diff
@@ -45,6 +45,24 @@ message ClientCmdID {
   optional int64 random = 2 [(gogoproto.nullable) = false];
 }
 
+// ReadConsistencyType specifies what type of consistency is observed
+// during read operations.
+enum ReadConsistencyType {
+  option (gogoproto.goproto_enum_prefix) = false;
+  // CONSISTENT reads are guaranteed to read committed data; the
+  // mechanism relies on clocks to determine lease expirations.
+  CONSISTENT = 0;
+  // CONSENSUS requires that reads must achieve consensus. This is a
+  // stronger guarantee of consistency than CONSISTENT.
+  //
+  // TODO(spencer): current unimplemented.
+  CONSENSUS = 1;
```

Comment:
But is this ever a choice you'd want to make on a per-operation basis? Either you trust your clocks and choose between CONSISTENT and INCONSISTENT, or you don't and choose between CONSENSUS and INCONSISTENT. The choice between CONSISTENT and CONSENSUS seems to be best made at the level of the deployment instead of the operation.